### PR TITLE
Link transactions page to sending bitcoin page

### PR DESCRIPTION
**Pull request by Bosch-0**
Minor PR adding a link from https://bitcoin.design/guide/payments/transactions/ to https://bitcoin.design/guide/payments/send/

### CODE REVIEW

File: guide/payments/transactions.md
Comment (johnsBeharry) [inline guide/payments/transactions.md:124]:
```suggestion

---

```

A horizontal rule is typically used to separate this statement from the above content. e.g. https://bitcoin.design/guide/getting-started/visual-language/

**Review by johnsBeharry — CHANGES_REQUESTED** [review]
Nice catch. You getting a bunch of the small details, we need this.

File: guide/payments/transactions.md
Comment (Bosch-0) [reply guide/payments/transactions.md:124]: It's used on all the other page to page links

Comment (johnsBeharry) [reply guide/payments/transactions.md:124]: The text is fine, just need the line spacer. Could you add it in? Should look like this.

Screenshot 2021-05-10 at 13 17 51

**Review by GBKS — APPROVED** [review]

**Review by pavlenex — APPROVED** [review]

**Comment by pavlenex** [comment]
@johnsBeharry waiting for your approval, the spacing issue has been addressed.

**Comment by pavlenex** [comment]
@johnsBeharry 🙏

**Review by johnsBeharry — APPROVED** [review]

**Pull request merged**
